prac_07/programming_language.py

```diff
@@ -0,0 +1,41 @@
+
+
+class ProgrammingLanguage:
+    """Represent information about a programming language."""
+
+    def __init__(self, name, typing, reflection, year, pointer_arithmetic):
+        """Construct a ProgrammingLanguage from the given values."""
+        self.name = name
+        self.typing = typing
+        self.reflection = reflection
+        self.year = year
+        self.pointer_arithmetic = pointer_arithmetic
+
```

Comment:
Good job! Clear, concise constructor with parameters that are easy to understand.